Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -691,6 +709,13 @@ class BrowserViewController: UIViewController {
     }
 }
 
+extension BrowserViewController: BluetoothKeyboardDelegate {
+    func openNewTab() {
+        urlBarDidPressTabs(self.urlBar)
+        NSNotificationCenter.defaultCenter().addObserver(self.tabTrayController, selector: "SELdidClickAddTab", name:BluetoothKeyboardNotification.TabTrayDidOpen, object: nil)
```

Comment:
Yikes, this is gnarly. Let's drop the `TabTrayController` middleman here (along with the observers) and handle this directly in `BrowserViewController`.